app/components/CV/section.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode } from 'react'
+
+type Props = {
+  name: string
+  data?: string[]
+  disclaimer?: {
+    companyName: string
+    referenceNumber: string
+  }
+  children?: ReactNode
+}
+
+const Section = ({ name, data = [], disclaimer, children }: Props) => {
+  return (
+    <div>
+      <h3 className="text-sm uppercase font-bold py-2">{name}</h3>
+      {children ? (
+        children
+      ) : (
+        <ul
+          className={`list-none text-xs ${name === 'portfolio' || name === 'kontakt' ? 'text-black' : 'text-slate-700'} `}
+        >
+          {disclaimer
```

Comment:
**suggestion**: Consider extracting the disclaimer text into a separate function or component for better readability and reusability.